lib/R.sc

```diff
@@ -1190,11 +1190,11 @@ RMultiOscillatorModule : RModule {
 	*params {
 		^[
 			'Range' -> (
-				Spec: ControlSpec.new(-2, 2, 'lin', 1, 0),
+				Spec: ControlSpec.new(-2, 2, nil, 1, 0),
 				LagTime: 0.01
 			),
 			'Tune' -> (
-				Spec: ControlSpec.new(-600, 600, 'lin', 0, 0, "cents"),
+				Spec: ControlSpec.new(-600, 600, nil, 0, 0, "cents"),
```

Comment:
'lin' -> nil - I believe these changes are unintended - it is is probably due to recent changes I made. Since this does not really apply to your changes, removing the changes would make the commit (and git blame) clearer.

Reply:
oh yea probably by accident - I might've messed with these then tried to change them back